packages/@aws-cdk/aws-ecr/lib/repository.ts

```diff
@@ -225,6 +274,19 @@ export interface OnCloudTrailImagePushedOptions extends events.OnEventOptions {
   readonly imageTag?: string;
 }
 
+/**
+ * Options for the OnImageScanCompleted method
+ */
+export interface OnImageScanCompletedOptions extends events.OnEventOptions {
+  /**
+   * Only watch changes to the image tags spedified.
+   * Leave it undefined to watch the full repository.
+   *
+   * @default undefined
```

Comment:
`undefined` is the literal default value, but tell us what the default *behavior* is when the value is left undefined.

Reply:
OK. I got it!